services/powershellgallery/powershellgallery.service.js

```diff
@@ -135,4 +140,78 @@ class PowershellGalleryDownloads extends BaseXmlService {
   }
 }
 
-module.exports = { PowershellGalleryVersion, PowershellGalleryDownloads }
+class PowershellGalleryPlatformSupport extends BaseXmlService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'powershellgallery/ps',
```

Comment:
Let's use `/platform` or `/p` here. I generally think the longer names are more self-documenting, though `/p` is what CocoaPods uses so we could also match that.

Reply:
changed